### ISSUE TITLE
[Stashing] Stashed changes view is not accessible 

### ISSUE THREAD

**Issue by iAmWillShepherd** [desktop/desktop#7274]
This issue was brought up in review of #7210. 

> If a changed file is present and you tab over to the Stashed Changes tab, clicking enter does not work. I expected doing so would allow me to view diffs.
> You cannot tab or key over to Stashed changes consistently, making it not accessible.
-- @tierninho 

![2019-04-12 11 34 43](https://user-images.githubusercontent.com/1715082/56052444-0d488c00-5d17-11e9-93c9-f557cd29abb9.gif)

☝️ I'm using <kbd>Tab</kbd> to cycle through these elements.
